applications/nrf_desktop/sample.yaml

```diff
@@ -22,47 +41,35 @@ tests:
       - nrf52840dongle/nrf52840
       - nrf52840gmouse/nrf52840
       - nrf5340dk/nrf5340/cpuapp
-      - nrf54h20dk/nrf54h20/cpuapp
-      - nrf54l15pdk/nrf54l15/cpuapp
-    tags: bluetooth ci_build sysbuild
   applications.nrf_desktop.zdebug_wwcb:
-    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
-    tags: bluetooth ci_build sysbuild
     extra_args: FILE_SUFFIX=wwcb
   applications.nrf_desktop.zdebug_fast_pair.gmouse:
-    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840 nrf52840gmouse/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf52840gmouse/nrf52840
-    tags: bluetooth ci_build sysbuild
     extra_args: FILE_SUFFIX=fast_pair
                 FP_MODEL_ID=0x8E717D
                 FP_ANTI_SPOOFING_KEY=dZxFzP7X9CcfLPC0apyRkmgsh3n2EbWo9NFNXfVuxAM=
   applications.nrf_desktop.zdebug_mcuboot_qspi:
-    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
-    tags: bluetooth ci_build sysbuild
     extra_args: FILE_SUFFIX=mcuboot_qspi
   applications.nrf_desktop.zdebug_mcuboot_smp:
-    sysbuild: true
```

Comment:
it would be great to apply this change for the whole NCS in one go or at least for our projects

Reply:
I could add it to our projects. I will do it under follow-up ticket: https://nordicsemi.atlassian.net/browse/NCSDK-28352. I think we could limit to our samples/apps (at least for now).